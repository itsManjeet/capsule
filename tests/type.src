println (type(10));
println (type(10.012));
println (type(true));
println (type(false));
println (type("Hello World"));
println (type('a'));
println (type(int));
println (type(float));
println (type(str));
println (type(bool));
println (type(type));
println (int == float);
println (int == int);
println (int == type (10));