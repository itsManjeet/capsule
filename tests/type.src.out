<type(int)>
<type(float)>
<type(bool)>
<type(bool)>
<type(str)>
<type(char)>
<type(type)>
<type(type)>
<type(type)>
<type(type)>
<type(type)>
false
true
true
